polynote-kernel/src/test/scala/polynote/testing/ZIOSpec.scala

```diff
@@ -16,11 +16,12 @@ import zio.{Runtime, ZIO}
 trait ZIOSpec extends Runtime[Clock with Console with System with Random with Blocking with Logging] {
   type Environment = Clock with Console with System with Random with Blocking with Logging
   // TODO: mock the pieces of this
-  val Environment: Environment =
+  override val environment: Environment =
     new Clock.Live with Console.Live with System.Live with Random.Live with Blocking.Live with Logging.Live
 
   // TODO: should test platform behave differently? Isolate per suite?
-  val Platform: Platform = PlatformLive.Default
+  override val platform: Platform = PlatformLive.Default
+    .withReportFailure(_ => ()) // suppress printing error stack traces by default
```

Comment:
@jeremyrsmith Also added this which suppresses printing stack traces of failed (but handled) effects